ROADMAP.md

```diff
@@ -248,6 +249,24 @@ rest of the system.
 
 - [NAT traversal tracking issue](https://github.com/libp2p/specs/issues/312).
 
+
+### ⏱ Full Observability
+
+**What?** libp2p should expose a wide set of metrics, making it easy to
+monitor the system.
+
+Metrics should include:
+- Transport metrics (TCP, QUIC, security protocols, stream multiplexers)
+- Swarm metrics
+- Resource Manager metrics
```

Comment:
```suggestion
```

I would categorize the "Resource Manager" a go-libp2p implementation detail. While rust-libp2p (partially) supports the same mechanism, it does not manifest as a separate component.

Reply:
I'm ok with removing it here, but this is a very important metric to track.